### PR TITLE
[4.0] use J4 namespacing

### PR DESCRIPTION
**Pull request by hans2103**
Partial Pull Request for Issue #33599 .

### Summary of Changes

This PR will change some J3 classNames by J4 namespaced classNames in Atum template

### Testing Instructions

just code change without visual change. 

### Actual result BEFORE applying this Pull Request

no visual changes to be expected.

### CODE REVIEW

**Comment by brianteeman** [comment]
please try not to use (or attempt to use) emoji in titles. 
At best they dont work and are meaningless 
At worse they break drone

**Comment by joomdonation** [comment]
It works but I'm unsure if the changed is needed. My my personal preference is keep using the fully qualified names for these lines of code. The reason is that it is only used in comment to help IDE to find the right class, so adding use statement is not necessary.

**Comment by hans2103** [comment]
Emoji removed.
I think JDocumentHtml and JDocumentError are J3. 
Since we're moving to J4 those classes need to be changed to HtmlDocument and ErrorDocument.
The reason to move it to `use` is be consistent with the rest. Just a way to write code... keeping all `use` stuff on one place.

**Comment by joomdonation** [comment]
> I think JDocumentHtml and JDocumentError are J3.

You are right and I agree with that. I just want to say that the class is not really used, it is used to help IDE only. So I would write it like this https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/error_full.php#L19 (save the system from having to run an unnecessary use command)

But as I said, it's not a big thing. It is just my own preference only.

**Comment by hans2103** [comment]
@joomdonation thank you for clear explanation. Used your suggestion
Please test my PR.

File: administrator/templates/atum/component.php [hunk 1]
```diff
@@ -9,9 +9,10 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Document\HtmlDocument;
 use Joomla\CMS\HTML\HTMLHelper;
 
-/** @var JDocumentHtml $this */
+/** @var HtmlDocument $this */
```
Comment (joomdonation) [inline administrator/templates/atum/component.php]: Could you please make the same change here, too? Thanks.

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 042de6640a061754c637d84568ec31934f320ea5<br /><br />Looks good now. Thanks :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33651">issues.joomla.org/tracker/joomla-cms/33651</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 042de6640a061754c637d84568ec31934f320ea5<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33651">issues.joomla.org/tracker/joomla-cms/33651</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33651">issues.joomla.org/tracker/joomla-cms/33651</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**